pkgs/data/icons/nordzy-icon-theme/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+, gtk3
+, nordzy-theme-name ? "Nordzy"
```

Comment:
```suggestion
```
Are you adding this argument because otherwise it uses the name of the derivation as the icon name (`share/icons/nordzy-icon-theme-unstable-2021-12-14`), or because you think it may useful to use a name different from the default name?

If the first, it suffices to unset the `name`environment variable before running `install.sh`, as shown below.

Reply:
I think it is useful to set the name, which is also the functionality from the upstream. I just thought it would be easiest to use with an argument.